integration-tests/tests/login_test.go

```diff
@@ -83,7 +83,7 @@ func (s *loginSuite) TestFakeServerIsDetected(c *check.C) {
 	err := s.writeCredentials(validLoginName)
 	c.Assert(err, check.IsNil, check.Commentf("error writting credentials"))
 
-	expectedMsg := fmt.Sprintf("Post https://%s/api/v2/tokens/oauth: x509: certificate is valid for example.com, not %s", loginHost, loginHost)
+	expectedMsg := fmt.Sprintf("Post https://%s/api/v2/tokens/discharge: x509: certificate is valid for example.com, not %s", loginHost, loginHost)
 	err = wait.ForFunction(c, expectedMsg, func() (string, error) { return s.stdout.String(), err })
 	c.Assert(err, check.IsNil, check.Commentf("didn't get expected fake server error"))
```

Comment:
Maybe: check.Commentf("didn't get the exepcted fake server error, but got: %s", err)? or something along these lines so that we know what we got instead of just that we did not get what we expected :)